### DISCUSSION TITLE
Are yaml anchors broken in flux 0.15?

### DISCUSSION THREAD

**Discussion by artyom-p** [fluxcd/flux2#1521] [Q&A] [answered]
Hi, just updated to flux 0.15, its mentioned that kustomize is updated to v4. After running updating flux my manifests doesnt work anymore :

error: node-referrerOriginal 'Namespace' mismatch 'foo' != 'bar'

**Comment by stefanprodan** [comment, marked as answer]
yaml anchors are broken in kustomize v4, see https://github.com/kubernetes-sigs/kustomize/issues/3675

**Reply by stefanprodan** [reply]
More breaking changes due to kustomize v4 here https://github.com/fluxcd/flux2/issues/1522
